src/components/facets/Facet.tsx

```diff
@@ -41,6 +42,7 @@ export interface IFacetDispatchProps {
 export interface IFacetChildrenProps {
     moreLabel?: string;
     filterPlaceholder?: string;
+    enabledExclude?: boolean;

```

Comment:
I think renaming `enabledExclude` for `excludeEnabled`, `exclusionsEnabled` or `enableExclusions` would make more sense